tmt/beakerlib.py

```diff
@@ -134,12 +135,25 @@ def fetch(self):
         # Initialize library cache (indexed by the repository name)
         if not hasattr(self.parent, '_library_cache'):
             self.parent._library_cache = dict()
-
         # Check if the library was already fetched
         try:
             library = self.parent._library_cache[self.repo]
             # The url must be identical
             if library.url != self.url:
+                # tmt guessed url so try if repo exists
+                if self.format == 'rpm':
+                    with TemporaryDirectory() as tmp:
+                        command = ['git', 'clone', '--depth=1', self.url,
+                                   tmp]
+                        try:
+                            self.parent.run(
+                                command, env={
+                                    "GIT_ASKPASS": "echo"})
+                        except tmt.utils.RunError as error:
```

Comment:
I expect flake8 to report `error` as an unused variable.

Reply:
good point, will fix and rebase to see what else flake8 finds

Reply:
you were right.